delius-int/sub-projects/new-tech.tfvars

```diff
@@ -50,3 +67,14 @@ offender_api_allowed_secure_cidrs = [
   "35.177.252.54/32",  # cloudplatform-live1-3
   "35.177.252.195/32"  # healthkick
 ]
+
+offender_search_allowed_secure_cidrs = [
+  "81.134.202.29/32",  # Moj VPN
```

Comment:
You can remove these since offender-searchis no longer used (it was moved to cloud platform). Probably needs clearing up from test etc as

Reply:
Have removed the values - can the config block be removed completely?